docs/pages/includes/config-reference/auth-service.yaml

```diff
@@ -339,6 +339,10 @@ auth_service:
     keep_alive_interval: 5m
     keep_alive_count_max: 3
 
+    # Specifies the dial timeout used for establishing SSH connections.
```

Comment:
I think it would be good to describe (or provide examples of) when the Auth Service would establish an SSH connection. Otherwise, it'll be difficult to tell when to adjust this value without talking to our support team.

Reply:
👍 Thanks! I'd specify that it's a cluster-wide setting and usually doesn't need to be adjusted